ldclient/client.py

```diff
@@ -35,36 +39,50 @@ def __init__(self,
                  verify=True,
                  defaults=None,
                  events=True,
-                 stream_processor_class=None,
-                 feature_store_class=None,
-                 feature_requester_class=None,
+                 update_processor_class=None,
+                 poll_interval=1,
+                 use_ldd=False,
+                 feature_store=InMemoryFeatureStore(),
+                 feature_requester_class=FeatureRequesterImpl,
                  consumer_class=None):
         """
 
-        :param stream_processor_class: A factory for a StreamProcessor implementation taking the api key, config,
+        :param update_processor_class: A factory for an UpdateProcessor implementation taking the api key, config,
                                        and FeatureStore implementation
-        :type stream_processor_class: (str, Config, FeatureStore) -> StreamProcessor
-        :param feature_store_class: A factory for a FeatureStore implementation
-        :type feature_store_class: () -> FeatureStore
+        :type update_processor_class: (str, Config, FeatureStore) -> UpdateProcessor
+        :param feature_store: A FeatureStore implementation
+        :type feature_store: FeatureStore
         :param feature_requester_class: A factory for a FeatureRequester implementation taking the api key and config
-        :type feature_requester_class: (str, Config) -> FeatureRequester
+        :type feature_requester_class: (str, Config, FeatureStore) -> FeatureRequester
         :param consumer_class: A factory for an EventConsumer implementation taking the event queue, api key, and config
         :type consumer_class: (queue.Queue, str, Config) -> EventConsumer
         """
         if defaults is None:
             defaults = {}
 
         self.base_uri = base_uri.rstrip('\\')
+        self.get_latest_features_uri = self.base_uri + GET_LATEST_FEATURES_PATH
         self.events_uri = events_uri.rstrip('\\')
         self.stream_uri = stream_uri.rstrip('\\')
-        self.stream = stream
-        self.stream_processor_class = RequestsStreamProcessor if not stream_processor_class else stream_processor_class
-        self.feature_store_class = InMemoryFeatureStore if not feature_store_class else feature_store_class
+        self.stream_features_uri = self.stream_uri + STREAM_FEATURES_PATH
+
+        if update_processor_class:
```

Comment:
you can manually set the update processor class (ie twisted)- if so, the stream param is ignored.